app/controllers/RepaymentAmountSummaryController.scala

```diff
@@ -45,6 +48,13 @@ class RepaymentAmountSummaryController @Inject()(
         Seq(helper.getTotalSection()).filter(_ => helper.getSections().size > 1)
       ).flatten
 
-      Ok(view(sections, mode))
+      val checkYourAnswersHelper = new CheckYourAnswersHelper(request.userAnswers)
+
+      try {
```

Comment:
can you replace this with scala Try

Reply:
and why is this throwing an exception in the first place. Signature of the method is: def getCheckYourAnswerSections: Seq[AnswerSection] . If you think it can throw an exception then change the places in this method where that can happen or wrap this in a Try.

Reply:
Ideally this method won't throw an error if we go through complete journey. But I am using this method to check whether RepaymentAmountSummary we are invoking from normal journey or from CYA. I have tried different ways to accomplish this, but I couldn't come across any better solution than this. Please advise if you have any.